Cargo.toml

```diff
@@ -85,7 +118,12 @@ tracing-subscriber = { version = "0.3.16", optional = true }
 derive_more = "0.99.17"
 rand = "0.8"
 tempfile = "3.2"
-tokio = { version = "1.12", features = ["macros", "rt-multi-thread", "sync", "time"] }
+tokio = { version = "1.12", features = [
+    "macros",
+    "rt-multi-thread",
+    "sync",
+    "time",
+] }
```

Comment:
@barafael I've found it more comfortable to keep each dependency as a single line in `[dependencies]` section.